app/models/auth_token.rb

```diff
@@ -67,13 +67,10 @@ def self.new_client(token, options = {})
   def self.new_v4_client(token)
     token ||= AuthToken.token
     http_adapter = GraphQL::Client::HTTP.new("https://api.github.com/graphql") do
-      @token = token
-
+      @@token = token # rubocop: disable Style/ClassVars
       # rubocop: disable Lint/NestedMethodDefinition:
       def headers(_context)
-        {
-          "Authorization" => "Bearer #{@token}",
-        }
+        { "Authorization" => "bearer #{@@token}" }
```

Comment:
Since this is a class variable now won't this mean that all v4 clients end up using the last token that was defined in here? What is the scope of this block and do we need to keep the token around after defining this `headers` method? For example could this just be `{ "Authorization" => "bearer #{token}" }`?